src/oss-tests/FindSourceTests.cs

```diff
@@ -62,15 +63,17 @@ public async Task Check_Sarif(string purl, string targetResult)
                     Kind = ResultKind.Informational,
                     Level = FailureLevel.None,
                     Rank = confidence,
-                    Locations = OutputBuilder.BuildPurlLocation(new PackageURL(purl))
+                    Locations = SarifOutputBuilder.BuildPurlLocation(new PackageURL(purl))
                 };
 
                 sarifResults.Add(sarifResult);
             }
 
-            OutputBuilder outputBuilder = new OutputBuilder("sarifv2");
+            IOutputBuilder outputBuilder = OutputBuilderFactory.CreateOutputBuilder("sarifv2");
             outputBuilder.AppendOutput(sarifResults);
-            SarifLog sarif = outputBuilder.BuildSingleRunSarifLog();
+            string sarifJSON = outputBuilder.GetOutput() ?? "{};";
+            SarifLog sarif = JsonConvert.DeserializeObject<SarifLog>(sarifJSON);
```

Comment:
GetOutput is no longer defined as returning null so this null coalescing operator is not going to work.